package.json

```diff
@@ -59,7 +59,8 @@
     "@backstage/errors": "workspace:^",
     "@changesets/changelog-github": "^0.4.8",
     "@manypkg/get-packages": "^1.1.3",
-    "@useoptic/optic": "^0.50.10"
+    "@useoptic/optic": "^0.50.10",
+    "lodash": "^4.17.21"
```

Comment:
You have to install `lodash` on the plugin itself rather than on the root. Each plugin is it's own package & can be bundled & published seperately. Our project setup will ensure the change is reflected in the `yarn.lock` file correctly.

Reply:
I tried to add the package, but get an error message unfortunately.
> yarn --cwd plugins/auth-backend-module-microsoft-provider add lodash
**canvas@npm:2.11.2 couldn't be built successfully**
The library for Mac can't be downloaded, so I will try Windows instead.

Reply:
Now it worked and the PR is complete. Unfortunately some checks are broken in nearly all PRs.